tests/business/utils/test_dataset_utils.py

```diff
@@ -328,20 +328,21 @@ def test_get_dummy_and_dependent_var_list(self):
         """Test if you receive the name of the dummy and dependent variables variable
         in a ugrid dataset"""
         # Arrange
-        var_list = ["var1", "var2", "var3", "var4", "var5"]
+        var_list = ["mesh2d", "var2", "var3", "var4", "var5"]
```

Comment:
it should not matter what the name of the dummy var is right?

Reply:
no it does'nt, but this made it easier to check if it was correct